cc/algorithms/count_test.cc

```diff
@@ -43,11 +43,11 @@ class CountTestPeer {
 
 namespace {
 
-using ::differential_privacy::test_utils::ZeroNoiseMechanism;
```

Comment:
Could you undo sorting of include files here?

Reply:
Fixed in the next commit.